src/container_registry/scaleway_container_registry.rs

```diff
@@ -260,6 +282,17 @@ impl ScalewayCR {
 
         self.create_registry_namespace(&image)
     }
+
+    fn get_docker_json_config_raw(&self) -> String {
+        base64::encode(
+            format!(
+                "{{\"auths\":{{\"rg.{}.scw.cloud\":{{\"auth\":\"{}\"}}}}}}",
```

Comment:
OPT: You can use raw string to avoid all the escapes

Reply:
Done